service/controller/v2/resources/vaultcrt/current.go

```diff
@@ -49,6 +50,13 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	// draining was done and the pods got removed we get an empty list here after
 	// the delete event got replayed. Then we just remove the secrets as usual.
 	if key.IsDeleted(customObject) {
+		// If this customObject is not the cert we are supporting in certs library,
+		// we don't need to check for running pods.
+		if !r.checkCertType(customObject) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", "unsupported cert type")
```

Comment:
```suggestion
			r.logger.LogCtx(ctx, "level", "debug", "message", "unsupported cert type %#q", key.ClusterComponent(customObject))
```

I think we should log the component so it's clear what is going on.

Reply:
@rossf7 Thanks, I added unittest as well.